src/app/pages/signup/signup.page.ts

```diff
@@ -28,11 +27,11 @@ export class SignupPage implements OnInit {
   constructor(private router: Router, private activatedRoute: ActivatedRoute, public formBuilder: FormBuilder, private authService: AuthService, private googleAuthService: GoogleAuthService, private linkedinAuthService: LinkedinAuthService) {
     this.registerForm = new FormGroup({
       "mobile": new FormControl('', Validators.compose([
-        Validators.pattern('[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]{1,10}$'),
+        Validators.pattern('^[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]+[0-9]{1,10}$'),
         Validators.maxLength(10),
         Validators.minLength(10)
       ])),
-      "email": new FormControl('', [Validators.pattern('[a-z0-9._%+-]+@[a-z0-9.-]+.[a-z]{2,3}$')])
+      "email": new FormControl('', [Validators.pattern('^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+[.][a-zA-Z]{2,}$')])
```

Comment:
The original email pattern here was actually pretty good but I made some tweaks like not limiting the root level length to a maximum of three characters and allowing capital letters.